src/managers/ReactionUserManager.js

```diff
@@ -1,16 +1,17 @@
 'use strict';
 
-const BaseManager = require('./BaseManager');
+const CachedManager = require('./CachedManager');
 const { Error } = require('../errors');
 const Collection = require('../util/Collection');
 
 /**
  * Manages API methods for users who reacted to a reaction and stores their cache.
- * @extends {BaseManager}
+ * @extends {CachedManager}
  */
-class ReactionUserManager extends BaseManager {
-  constructor(client, iterable, reaction) {
-    super(client, iterable, { name: 'User' });
+class ReactionUserManager extends CachedManager {
+  constructor(reaction, iterable) {
+    super(reaction.client, { name: 'User' }, iterable);
```

Comment:
Why isn't `User` used?

Reply:
In both cases the changing PR does not explain why this change was made:
- https://github.com/discordjs/discord.js/pull/4616
- https://github.com/discordjs/discord.js/pull/3696

There doesn't seem to be a circular require issue and nothing seems to break when using the appropriate classes instead of this dummy object.

I suppose it's fine to supply the classes again then, although this is probably outside of the scope of this PR.